chapter_1/exercise_1_13/histogram.c

```diff
@@ -28,9 +27,8 @@ int main(void)
   {
     if (c == ' ' || c == '\t' || c == '\n')
     {
-      if (!new_word)
+      if (word_count_index > 0)

```

Comment:
@DenSel86 This is a very nice optimization. Great job! 💪

Reply:
Thanks :)